geaflow/geaflow-dsl/geaflow-dsl-connector/geaflow-dsl-connector-hbase/src/main/java/com/antgroup/geaflow/dsl/connector/hbase/HBaseConfigKeys.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2023 AntGroup CO., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.antgroup.geaflow.dsl.connector.hbase;
+
+import static com.antgroup.geaflow.dsl.connector.hbase.HBaseConstants.DEFAULT_BUFFER_SIZE;
+import static com.antgroup.geaflow.dsl.connector.hbase.HBaseConstants.DEFAULT_FAMILY_MAPPING;
+import static com.antgroup.geaflow.dsl.connector.hbase.HBaseConstants.DEFAULT_NAMESPACE;
+import static com.antgroup.geaflow.dsl.connector.hbase.HBaseConstants.DEFAULT_SEPARATOR;
+
+import com.antgroup.geaflow.common.config.ConfigKey;
+import com.antgroup.geaflow.common.config.ConfigKeys;
+
+public class HBaseConfigKeys {
+
+    public static final ConfigKey GEAFLOW_DSL_HBASE_ZOOKEEPER_QUORUM = ConfigKeys
+        .key("geaflow.dsl.hbase.zookeeper_quorum")

```

Comment:
The config key format should be unified into `a.b.c`, please change the `_` to `.`

Reply:
fixed.